### PR TITLE
Update sshpass to 1.09

### PR DESCRIPTION
**Pull request by neha170**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 

Update sshpass to 1.09

###### Change Log  
- Update sshpass to 1.09

###### Does this affect the toolchain?  
NO

###### Associated issues  
- None

###### Links to CVEs  
- None

###### Test Methodology
- Local build

### CODE REVIEW

**Review by christopherco — COMMENTED** [review]
Upgrade itself looks good. Just a nit pick - Looks like spec linter is flagging some changes. Please address those as well.

File: SPECS/sshpass/sshpass.spec [hunk 1]
```diff
@@ -1,13 +1,13 @@
-Summary:	Noninteractive ssh password provider
-Name:		sshpass
-Version:	1.09
-Release:       1%{?dist}
-License:	GPLv2+
-URL:		http://sourceforge.net/projects/sshpass/
-Source0:	http://downloads.sourceforge.net/project/sshpass/%{name}/%{version}/%{name}-%{version}.tar.gz
-Group:		Applications/Networking
+Summary:	    Noninteractive ssh password provider
```
Comment (jslobodzian) [inline SPECS/sshpass/sshpass.spec:1]: tabs vs spaces?  I think the alignment is off here.  Can you make sure this is all spaces?

File: SPECS/sshpass/sshpass.spec [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Summary:	Noninteractive ssh password provider
 Name:		sshpass
-Version:	1.06
-Release:        4%{?dist}
+Version:	1.09
+Release:       1%{?dist}
```
[the remark this answers is not in the document]
Comment (neha170) [reply SPECS/sshpass/sshpass.spec]: Done, thanks Chris

File: SPECS/sshpass/sshpass.spec (hunk 1 above)
Comment (neha170) [reply SPECS/sshpass/sshpass.spec]: Yes, it was the tabs, thanks Jon

**Review by jslobodzian — APPROVED** [review]

**Pull request merged**